projects/canopy/src/lib/utils/dynamic-style.service.ts

```diff
@@ -0,0 +1,70 @@
+import { Injectable } from '@angular/core';
+
+import { BreakpointValues } from '../shared/breakpoints.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+/**
+ * Dynamically add styles to a <style> tag in <head>
+ */
+export class DyanmicStyleService {
```

Comment:
I know it's a draft but the typo :-) `DyanmicStyleService => DynamicStyleService`

Reply:
Changed